src/Appwrite/Platform/Workers/Hamster.php

```diff
@@ -0,0 +1,434 @@
+<?php
+
+namespace Appwrite\Platform\Workers;
+
+use Appwrite\Event\Hamster as EventHamster;
+use Appwrite\Network\Validator\Origin;
+use Utopia\Analytics\Adapter\Mixpanel;
+use Utopia\Analytics\Event as AnalyticsEvent;
+use Utopia\App;
+use Utopia\Cache\Cache;
+use Utopia\CLI\Console;
+use Utopia\Config\Config;
+use Utopia\Platform\Action;
+use Utopia\Database\Database;
+use Utopia\Database\Query;
+use Utopia\Database\Validator\Authorization;
+use Utopia\Database\Document;
+use Utopia\Queue\Message;
+use Utopia\Logger\Log;
+use Utopia\Pools\Group;
+
+class Hamster extends Action
+{
+    private array $metrics = [
+        'usage_files' => 'files.$all.count.total',
+        'usage_buckets' => 'buckets.$all.count.total',
+        'usage_databases' => 'databases.$all.count.total',
+        'usage_documents' => 'documents.$all.count.total',
+        'usage_collections' => 'collections.$all.count.total',
+        'usage_storage' => 'project.$all.storage.size',
+        'usage_requests' => 'project.$all.network.requests',
+        'usage_bandwidth' => 'project.$all.network.bandwidth',
+        'usage_users' => 'users.$all.count.total',
+        'usage_sessions' => 'sessions.email.requests.create',
+        'usage_executions' => 'executions.$all.compute.total',
+    ];
+
+    protected Mixpanel $mixpanel;
```

Comment:
Can/should this be Adapter instead and then we use an adapter based on env var?

```suggestion
    protected Utopia\Analytics\Adapter $analytics;
```

Reply:
Currently we only want to export this to mixpanel. The possibility of using another provider at the moment is very slim